### ISSUE TITLE
vignette for missing strata

### ISSUE THREAD

**Issue by JaneSullivan-NOAA** [afsc-assessments/rema#17]
Add a vignette on how to fill in missing strata using rema when developing a design-based index of abundance (e.g., for use in an age-structured model). Example: GOA dover sole

@Cole-Monnahan-NOAA, in this case would be it be appropriate to use the predicted value SD of the missing strata (converted to the natural scale) when summed with the other design-based variance estimates? Also pls feel free to add thoughts or other ideas for vignettes!

**Comment by Cole-Monnahan-NOAA** [comment]
Yes I prefer that approach of filling in missing strata estimates with the rema output. A dover sole example would be welcome for sure.

**Comment by careymcg** [comment]
For a future vignette (though variance calculations need to be double-checked): 

`# stitch design based strata with rema where there are design-based gaps:
surv_years<-unique(new_data$Year)
rema_by_strata_2023<-output$biomass_by_strata %>%
  mutate(obs_var = (obs_cv*obs)^2,pred=exp(log_pred),pred_var = (exp(log_pred) * (sqrt(exp(sd_log_pred^2)-1)))^2) %>%
  select(c(year,strata,pred,pred_var,obs,obs_var)) %>%
  filter(year %in% surv_years) %>%
  mutate(index = coalesce(obs,pred),index_var = coalesce(obs_var,pred_var)) %>%
  group_by(year)

index_total<-rema_by_strata_2023 %>% group_by(year) %>% summarise(tot_index = sum(index),tot_index_var = sum(index_var)) %>%
  mutate(tot_index_cv = sqrt(tot_index_var)/tot_index) %>%
  mutate(tot_index_log_se = sqrt(log(tot_index_cv^2+1)))`
